### PR TITLE
feat(atst): Add a gif of the cli to the readme

### PR DESCRIPTION
**Pull request by roninjin10**
- add gif of cli to readme
- commit the recording so we can rerecord

preview

### CODE REVIEW

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5034** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5034" target="_blank">Graphite</a> 
    * **PR #5036** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5036" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5036?utm_source=stack-comment).

File: packages/atst/assets/preview.tape [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+Type "atst write --key attitude --about 0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045 --value 'feeling very optimistic' --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 --rpc-url http://localhost:8545"
+Enter
+Sleep 2000ms
+
+Type "atst read --key attitude --about 0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045 --creator 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 --rpc-url http://localhost:8545"
+Enter
+Sleep 2000ms
+
+Type "atst write --key impress-level --about 0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045 --value 10 --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 --rpc-url http://localhost:8545"
```
Comment (roninjin10) [inline packages/atst/assets/preview.tape:58]: this private key is just the first anvil test address

File: packages/atst/docs/cli.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # atst cli docs
 
+![preview](https://user-images.githubusercontent.com/35039927/222435290-0271bc85-1e62-4d0a-b539-084af1e22ded.gif)
```
Comment (clabby) [inline packages/atst/docs/cli.md:3]: Probably best to do a relative link here
```suggestion
![preview](../assets/preview.gif)
```

File: packages/atst/README.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ The typescript sdk provides a clean [wagmi](https://wagmi.sh/) based interface f
 
 The cli provides a convenient cli for interacting with the attestation station contract
 
-TODO put a gif here of using it
+![preview](https://user-images.githubusercontent.com/35039927/222435290-0271bc85-1e62-4d0a-b539-084af1e22ded.gif)
```
Comment (clabby) [inline packages/atst/README.md:41]: Same here
```suggestion
![preview](./assets/preview.gif)
```

**Review by clabby — COMMENTED** [review]
Awesome!

File: packages/atst/docs/cli.md (hunk 2 above)
Comment (roninjin10) [reply packages/atst/docs/cli.md:3]: Oh good idea then we update these automatically when the gif changes

**Review by clabby — APPROVED** [review]

**Review by qbzzt — APPROVED** [review]

**Review by jakim929 — APPROVED** [review]
damn the vhs package is pretty cool 😎

File: packages/atst/docs/cli.md (hunk 2 above)
Comment (roninjin10) [reply packages/atst/docs/cli.md:3]:
```suggestion
![preview](../assets/preview.gif)
```
Force pushing accidentally wiped these changes

**Pull request merged**